external_deps/build.sh

```diff
@@ -716,12 +714,12 @@ setup_mingw64() {
 }
 
 # Set up environment for Mac OS X 64-bit
-setup_macosx64() {
+setup_macos-amd64-default() {
 	HOST=x86_64-apple-darwin11
 	CROSS=
 	MSVC_SHARED=(--disable-shared --enable-static)
 	export MACOSX_DEPLOYMENT_TARGET=10.9
-	export NASM="${PWD}/build-${PLATFORM}-${DEPS_VERSION}/prefix/bin/nasm" # A newer version of nasm is required for 64-bit
+	export NASM="${PWD}/${BUILD_BASEDIR}/prefix/bin/nasm" # A newer version of nasm is required for 64-bit
```

Comment:
I hit a bug here - the definition of BUILD_BASEDIR comes from common_setup. You can fix it by moving this line to the bottom of the function.

Reply:
Done.